tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -132,14 +134,14 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
     compat = Map(
       "3" ->
         // scala 3 type completions not implemented, so no way to distinguish if we are at a type position
-        """|IndexedSeq
-           |IndexedSeq
+        """|IndexedSeq[$0]
            |""".stripMargin
     ),
   )
 
   checkSnippet(
-    "type2".tag(IgnoreScala3),
+    "type2"
```

Comment:
It should actually. Let me check that.

Reply:
It requires implementing the case for `CompletionKind.Members` in `enrichWithSymbols`, since nothing is returned from the compiler.
So it can get done in this PR or maybe handled in a new one instead?

Reply:
update: I'm already working on it. It turns out that it is very much similar to handling `CompletionKind.Scope`

Reply:
update: It requires separate substantial work in filtering non-member suggestions, which also cause failure in the currently passing tests. Better be delegated to a followup PR, IMHO, because the code for it should go in `enrichWithSymbols` which is not currently touched in this PR.

Reply:
Do we have an issue about it? If not, could you raise it?

Reply:
https://github.com/scalameta/metals-feature-requests/issues/292 I wanted to self-assign but not having permission